recipe/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "vinecopulib" %}
+{% set version = "0.6.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d16c906f632f96e46112400225b10c4b28532daa3c35a97208881344a88c03cc
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - libboost {{ boost }}
+    - eigen 3.4.0
+    - wdm 0.2.4
+  run:
+    - libboost {{ boost }}
+    - eigen 3.4.0
+    - wdm 0.2.4
```

Comment:
@JeanChristopheMorinPerso is having these 3 deps in `run` a good idea? The headers depend anyway on those.

Reply:
You need all 3 because of:
* https://github.com/vinecopulib/vinecopulib/blob/258eb2d7758ce067ee7d53789067f0d3ebbb9375/cmake/findDependencies.cmake#L2
* https://github.com/vinecopulib/vinecopulib/blob/258eb2d7758ce067ee7d53789067f0d3ebbb9375/cmake/findDependencies.cmake#L4
* https://github.com/vinecopulib/vinecopulib/blob/258eb2d7758ce067ee7d53789067f0d3ebbb9375/cmake/findDependencies.cmake#L24

Reply:
then we are good, thanks!